src/config/internal.cpp

```diff
@@ -8,9 +9,37 @@
 
 #include <iterator>
 #include <optional>
+#include <oxen/log/format.hpp>
+
+using namespace oxen::log::literals;
 
 namespace session::config {
 
+namespace {
+
+    constexpr std::array<session::SessionIdPrefix, 6> all_session_id_prefixes = {
```

Comment:
The type and size should be auto-deducible here:
```suggestion
    constexpr std::array all_session_id_prefixes = {
```